lib/app/components/nuxt.js

```diff
@@ -7,6 +7,9 @@ export default {
   name: 'nuxt',
   props: ['nuxtChildKey'],
   render(h) {
+    if (this.nuxt._redirected) {
+      return h('div', [ 'redirecting.' ])
```

Comment:
Maybe we can support for i18n of this message :) I will handle it

Reply:
What about of  moving it to `options.messages` ? If ok, I will open a pr 😄

Reply:
If you can do it it would be awesome 😍